gormsupport/lifecycle.go

```diff
@@ -10,9 +10,9 @@ import (
 // The Lifecycle struct contains all the items from gorm.Model except the ID field,
 // hence we can embed the Lifecycle struct into Models that needs soft delete and alike.
 type Lifecycle struct {
-	CreatedAt time.Time
-	UpdatedAt time.Time
-	DeletedAt *time.Time
+	CreatedAt time.Time  `yaml:"created_at"`
```

Comment:
why do you need yaml tags here ? do we really need those attributes in the YAML templates ?